api/src/main/java/com/okta/sdk/client/AuthorizationMode.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.client;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Enumeration that defines the mapping between available Authentication schemes & Authorization modes.
+ */
+public enum AuthorizationMode {
+
+    SSWS(AuthenticationScheme.SSWS), // SSWS
+    PrivateKey(AuthenticationScheme.OAUTH2), // OAuth2
```

Comment:
This should probably follow java constant naming, maybe something like `PRIVATE_KEY("PrivateKey", AuthenticationScheme.OAUTH2)`, and then add static method in the enum to located it based on the string? (similar to t`Enum.valueOf`) 🤔

Reply:
Done.